dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/outbound/HttpOutputStreamImpl.java

```diff
@@ -740,4 +739,33 @@ public void setWC_remoteUser(String remoteUser) {
         }
     }
 
+    /**
+    * Handles MessageSentException based on ignoreWriteAfterCommit configuration
+    * @param mse The MessageSentException that was caught
+    * @throws IOException If the exception should be propagated
+    */
+    private void handleMessageSentException(MessageSentException mse) throws IOException {
+        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+            Tr.debug(tc, "Handling: MessageSentException " + this);
+        }
+
+        final HttpChannelConfig config = ((HttpInboundServiceContextImpl)this.isc).getHttpConfig();
```

Comment:
to avoid possible casting issues, consider enclosing this logic in a `instanceof` block